File: test/export/php/05-para.php
<?php
	$a = array();
	for ($i=1;$i<=10;$i++) {
		$a[$i] = $i*10;
	}
	foreach ($a as $elemento) {
		echo $elemento,PHP_EOL;
	}
	$b = array();
	$c = 0;
	foreach ($b as $aux_var_1) {
		foreach ($aux_var_1 as $x) {
			$c = $c+1;
			$x = $c;
		}
	}
	for ($i=3;$i>=1;$i--) {
		for ($j=1;$j<=5;$j+=2) {
			echo $b[$i][$j],PHP_EOL;
		}
	}
?>

File: test/export/c/05-para.c
#include<stdio.h>
int main() {
	float a[10];
	float b[3][6];
	float c;
	int i;
	int j;
	for (i=1;i<=10;i+=1) {
		a[i-1] = i*10;
	}
	for (typeof(&(a[0])) elemento=&(a[0]);elemento<=&(a[9]);elemento++) {
		printf("%s\n",(*elemento));
	}
	c = 0;
	for (typeof(&(b[0][0])) x=&(b[0][0]);x<=&(b[2][5]);x++) {
		c = c+1;
		(*x) = c;
	}
	for (i=3;i>=1;i-=1) {
		for (j=1;j<=5;j+=2) {
			printf("%f\n",b[i-1][j-1]);
		}
	}
	return 0;
}
